### PR TITLE
Fix/raid registering drive

### PR DESCRIPTION
**Pull request by armanddidierjean**
### Description

Please explain the changes you made here.

### Checklist

- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the documentation, if necessary

### CODE REVIEW

**Review by Malem38 — APPROVED** [review]

File: app/modules/raid/endpoints_raid.py [hunk 1]
```diff
@@ -531,8 +559,9 @@ async def set_security_file(
     if participant.security_file_id:
         # The participant already has a security file
         # We want to delete it to replace it by the new one
-        await cruds_raid.delete_security_file(
+        await cruds_raid.update_security_file(
```
Comment (guillaumetavernier) [inline app/modules/raid/endpoints_raid.py:562]: Is the comment still up to date ?

File: app/modules/raid/utils/drive/drive_file_manager.py [hunk 2]
```diff
@@ -1,162 +1,101 @@
...
-        self.drive_service = build(
-            "drive",
-            "v3",
-            credentials=oauth_credentials,
-            developerKey=settings.RAID_DRIVE_API_KEY,
-        )
-
+    def __init__(self):
         self.REGISTERING_FOLDER_NAME = "Équipes"
         self.SECURITY_FOLDER_NAME = "Fiches sécurité"
```
Comment (guillaumetavernier) [inline app/modules/raid/utils/drive/drive_file_manager.py:16]: We could also define constants as class attributes outside of __init__

File: app/modules/raid/utils/drive/drive_file_manager.py [hunk 3]
```diff
@@ -1,162 +1,101 @@
...
         self.drive_folders: coredata_raid.RaidDriveFolders | None = None
 
-    async def init_folders(self, db: AsyncSession):
-        if not self.drive_folders:
+    async def init_folders(self, db: AsyncSession, settings: Settings) -> None:
+        if self.drive_folders:
+            hyperion_error_logger.info(
+                "Raid Registering: drive folders already initialized",
+            )
```
Comment (guillaumetavernier) [inline app/modules/raid/utils/drive/drive_file_manager.py:23]: Couldn't we return here and remove the nested code from the `else` ?

File: app/modules/raid/utils/utils_raid.py [hunk 4]
```diff
@@ -170,14 +182,29 @@ async def post_update_actions(
...
-    if team:
-        if team.validation_progress == 100:
-            await set_team_number(team, db)
-            all_teams = await cruds_raid.get_all_validated_teams(db)
-            if all_teams:
-                await write_teams_csv(all_teams, db, drive_file_manager)
-        await save_team_info(team, db, drive_file_manager)
+    try:
+        if team:
+            if team.validation_progress == 100:
```
Comment (guillaumetavernier) [inline app/modules/raid/utils/utils_raid.py:189]:
```python
if team and team.validation == 100```

**Review by Rotheem — APPROVED** [review]

**Pull request merged**